modules/model/src/main/scala/navigate/model/enums/AcFilter.scala

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package navigate.model.enums
+
+import lucuma.core.util.Enumerated
+
+enum AcFilter(val tag: String) extends Product with Serializable derives Enumerated {
```

Comment:
Proably we don't need `Product with Serializable` any longer

Reply:
Force of habit